.ci.yaml

```diff
@@ -230,7 +230,6 @@ targets:
     properties:
       tags: >
         ["framework","hostonly","shard"]
-    scheduler: luci
```

Comment:
`bringup: true` to validate first?

Reply:
Since flutter/flutter's ci.yaml changes a lot, i'd rather leave it blocking. I plan to use [the engine PR to validate before landing this one.](https://github.com/flutter/engine/pull/31679)